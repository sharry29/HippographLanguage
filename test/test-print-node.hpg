int main() {
	node<int:string> n1 = 500:"foo";
	node<string:bool> n2 = "bar":true;
	node<bool:string> n3 = false;
	node<int:int> n4 = 12345:6789;
	n1.print();
	print("");
	n2.print();
	print("");
	n3.print();
	print("");
	n4.print();
}