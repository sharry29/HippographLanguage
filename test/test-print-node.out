500:"foo"
"bar":true
false:null
12345:6789